symphony/bdk/core/client/api_client_factory.py

```diff
@@ -9,6 +13,25 @@
 from symphony.bdk.gen.api_client import ApiClient
 from symphony.bdk.gen.configuration import Configuration
 
+trace_id_context = ContextVar("trace_id_context")
```

Comment:
Is this `trace_id_context` available from anywhere in the code? 

As a developer, I'd potentially like to init value from an HTTP filter if my app is a web application